src/components/UserDashboard/SavedQueries/index.js

```diff
@@ -78,6 +99,19 @@ const mapStateToProps = state => {
   };
 };
 
+let descriptions = new Array(20);
```

Comment:
Why did you use a module variable here? It won't be set when the query returns, so there won't be a render unless something else triggers it.

Please move to the state of the component, or use redux to store the virtual studies info. We currently only store the information of the loaded study in the cohort-builder, but we could store them all now that there is a need for it. See `src\store\reducers\ui\participantEntityPage.js` for an example. Reducers in the `ui` folder are for specific parts of the UI, if we end-up re-using the information, we will move it to a general-purpose reducer later.